.pipelines/templates/deploy-aks-cluster.yml

```diff
@@ -41,17 +39,6 @@ steps:
         --name ${RESOURCE_GROUP}
 
       # set NODE_RESOURCE_GROUP for e2e test config
-      export NODE_RESOURCE_GROUP="MC_${RESOURCE_GROUP}_${RESOURCE_GROUP}_$(LOCATION)"
+      NODE_RESOURCE_GROUP="$(az aks show -g ${RESOURCE_GROUP} -n ${RESOURCE_GROUP} --query nodeResourceGroup -otsv)"
       echo "##vso[task.setvariable variable=NODE_RESOURCE_GROUP]${NODE_RESOURCE_GROUP}"
-
-      AKS_CLUSTER_VERSION=$(az aks show -g ${RESOURCE_GROUP} -n ${RESOURCE_GROUP} --query 'kubernetesVersion' -o tsv)
-      # set AKS_CLUSTER_VERSION for e2e test config
-      echo "##vso[task.setvariable variable=AKS_CLUSTER_VERSION]${AKS_CLUSTER_VERSION}"
     displayName: "Deploy an AKS cluster "
-
-  - script: |
-      echo "Installing kubectl..."
```

Comment:
Is `kubectl` installed by default in the upstream pool?

Reply:
That's right